tests/systems/systems_test.C

```diff
@@ -1256,6 +1257,78 @@ public:
     LIBMESH_ASSERT_FP_EQUAL(system.solution->l1_norm(), ref_l1_norm, TOLERANCE*TOLERANCE);
   }
 
+
+  void testSetSystemParameterOverEquationSystem()
```

Comment:
This test isn't currently being run.  You need to add a `CPPUNIT_TEST( testSystemParameterOverEquationSystems );` macro invocation with the others at the start of the class.